src/ansible_creator/resources/common/patterns/sample_pattern/README.md

```diff
@@ -0,0 +1,52 @@
+## Weather Forecasting Pattern
+
+### Description
+
+This pattern is designed to help get the weather forecast for a given location.
+It uses the `site.yml` playbook to retrieve the weather forecast for a specified
+location.
+
+### What This Pattern Covers
+
+- Retrieves the weather forecast for a given location.
+- Uses the `site.yml` playbook to perform the weather forecasting task.
+
+### Resources Created by This Pattern
+
+1. Project
+
+- Ensures that all relevant files and configurations are logically arranged,
+  facilitating easier maintenance and execution of automation tasks.
+- This project is used to organize and manage the weather forecasting task.
+
+2. Job Templates
+
+- Outline the necessary parameters and configurations to perform weather
+  forecasting task using the provided playbook.
+
+### How to Use
+
+1. Use Seed Red Hat Pattern Job
```

Comment:
This is the old seeding method. The new way will be to use the pattern service to load the pattern; I'm not sure if we want to document use of that here.